x/ibc/handler.go

```diff
@@ -19,7 +19,8 @@ func NewHandler(k Keeper) sdk.Handler {
 			return client.HandleMsgCreateClient(ctx, k.ClientKeeper, msg)
 
 		case client.MsgUpdateClient:
-			return &sdk.Result{}, nil // XXX: add events either here or in ante
+			return &sdk.Result{}, nil
```

Comment:
Where are these events going to get added?

Reply:
The events have to be added in the antehandler

Reply:
we need to move the events from ICS02 handler to the `clientKeeper.UpdateClient` function instead. This hasn't been done. Unresolving

Reply:
OK, I did this (I think) in https://github.com/cosmos/cosmos-sdk/pull/5401/commits/f358a63c8e139f53382046ca5e0d6a8748e710aa